### PR TITLE
yq-go: init at 2.4.0

### PR DESCRIPTION
**Pull request by nlewo**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by nlewo** [comment]
@GrahamcOfBorg build yq-go

**Comment by nlewo** [comment]
@GrahamcOfBorg build yq-go

File: pkgs/development/tools/yq-go/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.3.0
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+# buildGoModule is not supported by the project
+# See https://github.com/mikefarah/yq/issues/227
+buildGoPackage rec {
+  name = "yq-go-${version}";
```
Comment (mmahut) [inline pkgs/development/tools/yq-go/default.nix]: Please use ``pname``.

Comment (nlewo) [reply pkgs/development/tools/yq-go/default.nix]: Done. Thx!

**Comment by mmahut** [comment]
@GrahamcOfBorg build yq-go

**Pull request merged**